src/providers/arcgisrest/qgsarcgisrestutils.cpp

```diff
@@ -38,6 +38,8 @@
 #include <QJsonDocument>
 #include <QJsonObject>
 
+#include <functional>
```

Comment:
I think this include is not needed, it will automatically be present because it has already been included in the header, no?